src/Microsoft.OpenApi/Validations/Visitors/VisitorBase.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.OpenApi.Interfaces;
+
+namespace Microsoft.OpenApi.Validations.Visitors
+{
+    /// <summary>
+    /// The base visitor class for Open Api elements.
+    /// </summary>
+    internal abstract class VisitorBase<T> : IVisitor where T : IOpenApiElement
+    {
+        /// <summary>
+        /// Visit the element itself.
+        /// </summary>
+        /// <param name="context">The validation context.</param>
+        /// <param name="item">The element.</param>
+        public void Visit(ValidationContext context, object item)
```

Comment:
>object [](start = 53, length = 6)

Can this be T?

Reply:
I tried, But i can't.

---
In reply to: [157650480](https://github.com/Microsoft/OpenAPI.NET/pull/152#discussion_r157650480) [](ancestors = 157650480)